apps/trading/components/welcome-dialog/risk-notice-dialog.tsx

```diff
@@ -88,29 +86,75 @@ const TestnetContent = ({ network }: { network: Networks }) => {
           </li>
         </ul>
       )}
+      <div className="my-4">
+        <TelemetryApproval
+          helpText={t(
+            'Help identify bugs and improve the service by sharing anonymous usage data. You can change this in your settings at any time.'
+          )}
+        />
+      </div>
+      <Button onClick={handleAcceptRisk}>{t('Continue')}</Button>
     </>
   );
 };
 
-const MainnetContent = () => {
+const MainnetContent = ({
+  handleAcceptRisk,
+}: {
+  handleAcceptRisk: () => void;
+}) => {
+  const updateStore = useGlobalStore((store) => store.update);
+  const { disconnect } = useVegaWallet();
+  const revokeWalletConnection = () => {
+    updateStore({ shouldDisplayMainnetRiskDialog: false });
+    disconnect();
+  };
+  const openDisclaimer = () => {
+    updateStore({ shouldDisplayDisclaimerDialog: true });
+  };
   return (
-    <>
-      <h4 className="text-xl mb-2 mt-4">
-        {t('Regulation may apply to use of this app')}
-      </h4>
-      <p className="mb-6">
-        {t(
-          'This decentralised application allows you to connect to and use publicly available blockchain services operated by third parties that may include trading, financial products, or other services that may be subject to legal and regulatory restrictions in your jurisdiction. This application is a front end only and does not hold any funds or provide any products or services. It is available to anyone with an internet connection via IPFS and other methods, and the ability to access it does not imply any right to use any services or that it is legal for you to do so. By using this application you accept that it is your responsibility to ensure that your use of the application and any blockchain services accessed through it is compliant with applicable laws and regulations in your jusrisdiction.'
-        )}
-      </p>
-      <h4 className="text-xl mb-2">
-        {t('Technical and financial risk of loss')}
-      </h4>
+    <div className="mt-6">
+      <div className="bg-neutral-100 dark:bg-neutral-800 dark:text-neutral-200 p-6 mb-6">
+        <ul className="list-disc ml-6 text-lg">
+          <li>
+            {t(
+              'No party hosts or operates this IFPS website or offers any financial advice.'
```

Comment:
This sentence has been cut so it would be worth reconsidering what goes in here

Reply:
text has been changed according given pattern